Core/src/IfcGeometryConverter/FaceConverter.h

```diff
@@ -801,59 +790,73 @@ namespace OpenInfraPlatform {
 					}
 				}
 				
-				void convertIfcFaceBoundList(
-					std::vector<EXPRESSReference<typename IfcEntityTypesT::IfcFaceBound>> ifcFaceBounds,
-					const carve::math::Matrix& pos,
-					std::vector<std::vector<carve::geom::vector<3>>>& faceBoundLoops) const noexcept(true)
+				std::vector<std::vector<carve::geom::vector<3>>> convertIfcFaceBoundList(
+					const std::vector<EXPRESSReference<typename IfcEntityTypesT::IfcFaceBound>>& ifcFaceBounds,
+					const carve::math::Matrix& pos) const noexcept(true)
+				{
+					const std::vector<EXPRESSReference<typename IfcEntityTypesT::IfcFaceBound>> faceBounds = swapOuterBoundaryToFront(ifcFaceBounds);
+
+					std::vector<std::vector<carve::geom::vector<3>>> faceBoundLoops;
+					faceBoundLoops.resize(faceBounds.size());
+					for (size_t i = 0; i < faceBounds.size(); i++)
+					{
+						// get vertices of one bound loop into faceBoundLoops[i]
+						faceBoundLoops[i] = convertIfcFaceBound(faceBounds[i], pos);
+					}

```

Comment:
you could use `std::transform` here (but not necessary to change; you can learn, however)

Reply:
Done in commit 7096ed8.